iphone/TitaniumKit/TitaniumKit/Sources/API/TiExceptionHandler.m

```diff
@@ -94,6 +94,7 @@ - (void)reportScriptError:(JSValue *)error inJSContext:(JSContext *)context
 
 - (void)showScriptError:(TiScriptError *)error
 {
+  // FIXME: If nativeStack is a string already this fails!
```

Comment:
This should be fixed in [#12155](https://github.com/appcelerator/titanium_mobile/pull/12155/files#diff-4aacf67647acf3dcda586104901c0f41add90eb30a18fa7cde15d8add6c460b6R1347)